pyteal/ast/subroutine.py

```diff
@@ -269,15 +292,39 @@ def has_return(self):
 SubroutineDeclaration.__module__ = "pyteal"
 
 
+@dataclass
+class _OutputKwArgInfo:
```

Comment:
I apologize for introducing underscored class names during #249. This was later removed and unless you have a good reason, I recommend removing here as well to adhere with the conventions of this repo.

Reply:
I removed the leading underline anyways.